src/components/2019/year-in-review/Sports.js

```diff
@@ -0,0 +1,267 @@
+import React from 'react'
```

Comment:
Generally this file has the same systemic nits and problems that News, Opinion and Sports have, if you make edits in line with the other sections you should be good (please request my review again when you're done so I can just check in to make sure)